ui/config/karma-test-shim.js

```diff
@@ -0,0 +1,21 @@
+Error.stackTraceLimit = Infinity;
```

Comment:
Yea I actually didn't see that, the testing setup was pretty much a copy paste from the example setup.